File: tests/scan/fixtures/javascript/01_basic_dependency_future/src/main.js
// @tear: 0
import { secret } from "./secret.js";
import "./side_effect.js";
export { exported } from "./z_exported.js";

const required = require("./z_required.js");

export const value = secret();

export async function load() {
  return [required, await import("./z_dynamic.js")];
}

File: tests/scan/fixtures/go/01_basic_dependency_future/src/main.go
// @tear: 0
package main

import (
	_ "example.com/tearsfixture/src/sideeffect"
	"example.com/tearsfixture/src/secret"
	aliassecret "example.com/tearsfixture/src/zalias"
	. "example.com/tearsfixture/src/zdot"
)

func main() {
	_ = secret.Value()
	_ = aliassecret.Value()
	_ = DotValue()
}
